encord/objects/ontology_labels_impl.py

```diff
@@ -1347,6 +1372,19 @@ def to_encord_dict(self) -> dict:
 
         return ret
 
+    def workflow_reopen(self) -> None:
+        """
+        Sets the label row for re-labeling by resetting its progress over the workflow graph
+        No data will be lost during this call.
+        This method only relevant for the projects that use the Encord Task Management System 2,
```

Comment:
"This method is only relevant for the projects that use Encord Task Management System 2"